src/middlewared/middlewared/plugins/service.py

```diff
@@ -903,7 +903,7 @@ def _started_notify(self, verb, what):
         await self._service("ix-post-samba", "start", quiet=True, **kwargs)
 
     async def _started_cifs(self, **kwargs):
-        if await self._service("samba_server", "status", quiet=True, onetime=True, **kwargs):
+        if await self._service("samba_server", "status", quiet=True, **kwargs):
```

Comment:
This was a changed introduced by me. It serves the purpose to correctly report if samba is running in case "start on boot" is not checked.

This will regress that bit.